src/config/dynamic-theme-fixes.config

```diff
@@ -3274,6 +3274,18 @@ CSS
 
 ================================
 
+feedly.com
+
+CSS
+.entry.u0:hover {
+    background-color: rgb(35, 35, 35) !important;
+}
+.entry--selected.u0, .entry--selected.u0:hover, button.secondary:hover {
```

Comment:
Please format it to
```CSS
.entry--selected, 
.entry--selected:hover, 
button.secondary:hover {
```
Please notice that the .u0 is redundant as AFAIK I have on every entry an `.u4` instead and doesn't specify any further.

Reply:
Actually, if I leave .u0 off it is not the same experience for me. Somehow, if you don't use .u0, the colours are being added up if you hover on a selected article so it becomes even brighter then just selected or just hover. That's two different colours then and that's exactly what we don't want to achieve.

Reply:
OK I found the differences, it showed `.u4` for you because you were in the "Magazine View". The "Cards View" has `.u5` and the "Article View" doesn't use an background at all. I use "Titles-Only View", which uses `.u0`

Change this in Settings > General.